compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenInterfaces.td

```diff
@@ -9,6 +9,146 @@
 
 include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenDialect.td"
 
+def IREECodegen_InnerTileDescAttrInterface :
+  AttrInterface<"InnerTileDescAttrInterface"> {
+  let cppNamespace = "::mlir::iree_compiler::IREE::Codegen";
+  let description = [{
+    Attribute interface used to descripe an inner tiled operation, such as
+    a (sequence of) matrix multiplications that are performed using
+    intrinsics which have their own inherent tiling.
+
+    Note: this interface is intended to generate MmaInterfaceAttr on a
+    case-by-case basis. The methods here are not currently complete, as we're
+    mid-refactoring. Feel free to add anything that'll be needed during the scaling
+    MMA project.
```

Comment:
I think we can leave off this comment, or simplify it to:
```suggestion
    Note: this interface is a generalization of MmaInterfaceAttr.
```
Referring to in progress work in the code is unusual, better to keep that in issues IMO (which you filed a great issue for already :) )